modules/negotiate/lib/Auth/Source/Negotiate.php

```diff
@@ -136,6 +138,12 @@ public function authenticate(&$state)
                     SimpleSAML\Logger::debug('Negotiate - authenticate(): No "Negotiate" found. Skipping.');
                 }
             }
+            
+            if ($this->rcache === true) {
+                putenv("KRB5RCACHETYPE=dfl");
+            } elseif ($this->rcache === false) {
```

Comment:
In that case, I think it makes sense to keep it like this then.

I would only add some documentation explaining the option, why is it necessary, what are its values, and what's the default. Other than that, 👍